system-addon/lib/TelemetryFeed.jsm

```diff
@@ -259,7 +259,7 @@ this.TelemetryFeed = class TelemetryFeed {
     const appInfo = this.store.getState().App;
     const ping = {
       addon_version: appInfo.version,
-      locale: appInfo.locale,
+      locale: Services.locale.getRequestedLocale(),
```

Comment:
How expensive is this check? Inexpensive enough that we can check on every ping created?

Reply:
Mmm... 
```js
s = performance.now(); for (i = 0; i < 1000000; i++) Services.locale.getRequestedLocale(); performance.now() - s`
1078.2949999999255
```
1 microsecond for me?

Reply:
sounds good to me 🚗